### ISSUE TITLE
Module doesn't link for Android

### ISSUE THREAD

**Issue by yolosaucisson** [callstack-io/react-native-fbads#37]
Hi,

I have React-Native v41.
I downloaded the react-native-fbads with yarn and after that I did "react-native link".
It showed me that the iOS module was link but nothing about the Android module.

How I have to do? Have I to install other things than this to make it working?

When I try to use it on my code, I can't use ith undefined with this line of code : 
import { NativeAdsManager } from 'react-native-fbads';
const adsManager = new NativeAdsManager('18025212...');

Sincerely,

**Comment by yolosaucisson** [comment]
When I try to use it on my code, I can't use ith undefined with this line of code : 
import { NativeAdsManager } from 'react-native-fbads';
const adsManager = new NativeAdsManager('18025212...');

**Comment by grabbou** [comment]
Hey,

If for some reason `react-native link` failed for you, please default to manual linking process as described in the official React documentation.

**Comment by yolosaucisson** [comment]
It failes for all people... with recent version so...

**Comment by grabbou** [comment]
You are right that there were already two reports regarding that matter, I'll reopen for now and debug later. If anyone wants to tackle it in the meantime, please do!

**Comment by yolosaucisson** [comment]
If you want I can help to be better in development. It could be an good point as a developer to upgrade a repo. But I don't really know for the moment how to do :/

**Comment by sfratini** [comment]
I found the issue. It seems the problem is that the react-native-fbads folder inside the node_module has one extra folder src which makes react-native miss the gradle file. 
I moved all the files inside the src folder (the one that has the android, ios folders) to the main one (react-native-fbads) and react-native link started working. 

Now I am finding another issue at compiling but I am not sure it it is related to this project yet.

**Comment by sfratini** [comment]
Alright, this is what I tried:

1) Update settings.gradle of your project and add this:
```
include ':react-native-fbads'
project(':react-native-fbads').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-fbads/src/android')
```

Note: At this point you probably need a Gradle Sync, Clean and Build for the project to come up as module. 

2) Modify the build.gradle file and in the dependencies, add this:
`compile project(':react-native-fbads')`

3) Modify the MainApplication.java and add these lines:
```
import io.callstack.react.fbads.FBAdsPackage;
new FBAdsPackage() in getPackages()
```

Now it compiles and builds but the APK is not generated due to an error in Duplicated definitions. Still trying to see what is going on.

**Comment by LutherDiaz** [comment]
@sfratini Try 
```
    compile(project(':react-native-fbads')) {
		exclude group: "com.google.android.gms"
    }
```

Not sure if this is a proper fix though.

**Comment by sfratini** [comment]
@LutherDiaz Yep, actually that is how I solved it. I was also getting the Dex too many methods error but I solved it with Proguard in debug.

**Comment by grabbou** [comment]
Fixed in 4475c36
